phoenix-core/src/main/java/org/apache/phoenix/query/QueryServicesOptions.java

```diff
@@ -284,14 +288,18 @@
     public static final int DEFAULT_COLUMN_ENCODED_BYTES = QualifierEncodingScheme.TWO_BYTE_QUALIFIERS.getSerializedMetadataValue();
     public static final String DEFAULT_IMMUTABLE_STORAGE_SCHEME = ImmutableStorageScheme.SINGLE_CELL_ARRAY_WITH_OFFSETS.toString();
     public static final String DEFAULT_MULTITENANT_IMMUTABLE_STORAGE_SCHEME = ImmutableStorageScheme.ONE_CELL_PER_COLUMN.toString();
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_CLUSTER_BASE_PATH = "/phoenix";
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_SERVICE_NAME = "queryserver";
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_ZK_ACL_USERNAME = "phoenixuser";
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_ZK_ACL_PASSWORD = "Xsjdhxsd";
```

Comment:
I think we should just change this to be something simple like "phoenix/phoenix" instead of "phoenixuser/Xsjdhxsd" (the public random text is no-more secure than an easily remember-able string).

We also need to document how users are expected to secure this (property to set in hbase-site.xml, make sure hbase-site.xml is not globally readable, etc). I'm seeing down below that there is not support for SASL ACLs -- that will also need to be documented as a limitation.